[2025-02-19T11:38:34.360+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-19T11:38:34.371+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.parser_for_Eapteka manual__2025-02-19T11:38:32.325643+00:00 [queued]>
[2025-02-19T11:38:34.377+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.parser_for_Eapteka manual__2025-02-19T11:38:32.325643+00:00 [queued]>
[2025-02-19T11:38:34.378+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-19T11:38:34.518+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): parser_for_Eapteka> on 2025-02-19 11:38:32.325643+00:00
[2025-02-19T11:38:34.527+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'parser_for_Eapteka', 'manual__2025-02-19T11:38:32.325643+00:00', '--job-id', '910', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp0cf_f354']
[2025-02-19T11:38:34.525+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=73) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-19T11:38:34.528+0000] {standard_task_runner.py:105} INFO - Job 910: Subtask parser_for_Eapteka
[2025-02-19T11:38:34.529+0000] {standard_task_runner.py:72} INFO - Started process 80 to run task
[2025-02-19T11:38:34.592+0000] {task_command.py:467} INFO - Running <TaskInstance: pipeline.parser_for_Eapteka manual__2025-02-19T11:38:32.325643+00:00 [running]> on host 23903a4a7553
[2025-02-19T11:38:34.664+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='parser_for_Eapteka' AIRFLOW_CTX_EXECUTION_DATE='2025-02-19T11:38:32.325643+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-19T11:38:32.325643+00:00'
[2025-02-19T11:38:34.665+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-19T11:38:34.666+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-19T11:38:34.666+0000] {logging_mixin.py:190} INFO - Current task name:parser_for_Eapteka state:running start_date:2025-02-19 11:38:34.371945+00:00
[2025-02-19T11:38:34.666+0000] {logging_mixin.py:190} INFO - Dag name:pipeline and current dag run status:running
[2025-02-19T11:38:34.667+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-19T11:38:34.667+0000] {parser_1.py:27} INFO - Open webdriver_remote
[2025-02-19T11:38:35.540+0000] {parser_1.py:39} INFO - Webdriver opened
[2025-02-19T11:39:42.043+0000] {parser_1.py:63} ERROR - Error occurred: Message: unknown error: net::ERR_NAME_NOT_RESOLVED
  (Session info: chrome=124.0.6367.78)
Stacktrace:
#0 0xaaaad1ef7d44 <unknown>
#1 0xaaaad1c1e374 <unknown>
#2 0xaaaad1c18a34 <unknown>
#3 0xaaaad1c0c274 <unknown>
#4 0xaaaad1c0d518 <unknown>
#5 0xaaaad1c0c5e4 <unknown>
#6 0xaaaad1c0bf64 <unknown>
#7 0xaaaad1c0be44 <unknown>
#8 0xaaaad1c0a4bc <unknown>
#9 0xaaaad1c0a7b8 <unknown>
#10 0xaaaad1c213ec <unknown>
#11 0xaaaad1c98848 <unknown>
#12 0xaaaad1c98400 <unknown>
#13 0xaaaad1c514a4 <unknown>
#14 0xaaaad1c520e0 <unknown>
#15 0xaaaad1ec5ac0 <unknown>
#16 0xaaaad1ec8a64 <unknown>
#17 0xaaaad1ec8630 <unknown>
#18 0xaaaad1eb6d6c <unknown>
#19 0xaaaad1ec9128 <unknown>
#20 0xaaaad1ea1010 <unknown>
#21 0xaaaad1ee8550 <unknown>
#22 0xaaaad1ee8758 <unknown>
#23 0xaaaad1ef7250 <unknown>
#24 0xffff948cee58 <unknown>
#25 0xffff94937f9c <unknown>

[2025-02-19T11:39:42.169+0000] {python.py:240} INFO - Done. Returned value was: Empty DataFrame
Columns: []
Index: []
[2025-02-19T11:39:42.244+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-19T11:39:42.245+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=parser_for_Eapteka, run_id=manual__2025-02-19T11:38:32.325643+00:00, execution_date=20250219T113832, start_date=20250219T113834, end_date=20250219T113942
[2025-02-19T11:39:42.267+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-19T11:39:42.267+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-19T11:39:42.268+0000] {logging_mixin.py:190} INFO - Dag name:pipeline queued_at:2025-02-19 11:38:32.343662+00:00
[2025-02-19T11:39:42.268+0000] {logging_mixin.py:190} INFO - Task hostname:23903a4a7553 operator:_PythonDecoratedOperator
[2025-02-19T11:39:42.309+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-19T11:39:42.336+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-19T11:39:42.337+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
